common/SC_Filesystem_unix.cpp

```diff
@@ -32,6 +32,11 @@
 // system
 #include <glob.h> // ::glob, glob_t
 
+// workaround for musl not yet supporting GLOB_TILDE
+#ifndef GLOB_TILDE
+#define GLOB_TILDE 0
```

Comment:
for our formatting standards we prefer to indent preprocessor directives like this:

```suggestion
#    define GLOB_TILDE 0
```